modules/apps/frontend-js/frontend-js-translation-web/src/main/resources/META-INF/resources/index.tsx

```diff
@@ -0,0 +1,30 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+import {render} from '@liferay/frontend-js-react-web';
+import React from 'react';
+
+import TranslationSelector, {IProps} from './TranslationSelector';
+
+declare global {
+	var Liferay: {
+		Language: {
+			get(value: string): string;
+		};
+	};
+}
+
+export default function (container: Element, props: IProps) {
+	render(<TranslationSelector {...props} />, {}, container);
+}
```

Comment:
Is the idea of directly calling `render` because we want to allow this component to also be used outside of taglib like `react:component`?

Reply:
When reading this code it brought this thing too but I forgot to make the question :p

Reply:
In dxp, we can't use `ReactDOM.render` directly (that throws an `eslint` error) , and yes we also want this component to be used directly with no taglib.

Reply:
Thanks @julien for info